py7zr/cli.py

```diff
@@ -274,6 +281,9 @@ def run_extract(self, args: argparse.Namespace) -> int:
         except py7zr.exceptions.Bad7zFile:
             print('Header is corrupted. Cannot read as 7z file.')
             return 1
+        except py7zr.exceptions.PasswordRequired:
+            print('The archive seems to be encrypted, and password is required...aborted.')
```

Comment:
it should show consistent warning message.
'The archive is encrypted but password is not given. FAILED.'

Reply:
An exception may raise not only when create SevenZipFile object, also extractall method.